Cargo.toml

```diff
@@ -9,15 +9,22 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/nannou-org/nannou_timeline.git"
 homepage = "https://nannou.cc"
 
+[features]
+default = ["serde"]
```

Comment:
I think maybe this would be better to follow the convention of the dependencies and make serde an opt-in feature rather than opt-out.

Reply:
That's probably true! I can't find an easy way of adding an optional package as a dev-dependency though so making serde optional means I have to run the demo using `cargo run --release --example demo --features "serde"`. Do you know of an easier way? Otherwise I can modify the demo to only use serde if available!